### DISCUSSION TITLE
AMD Radeon Pro VII - improvement with ROCm 5.0

### DISCUSSION THREAD

**Discussion by selroc** [preda/gpuowl#249] [Show and tell]
The PRP timing of exponent 114837847 went from 863 us/it to 854 us/it.

**Comment by preda** [comment]
Thanks; I also checked with a wavefront exponent (109M), and what I see is a slight speed-up in single-instance-per-GPU (which is good!), but at the same time about 4% slowdown in duo-instance-per-GPU (which is not good). (this is all relative to ROCm 3.3.0). So there is progress in some situation at least.

**Reply by selroc** [reply]
The improvement with ROCm 5.2.0 is notable:
for exponent 113396317: Radeon VII - 738 us/it
for exponent 110556109: Radeon Pro VII - 812 us/it

**Comment by selroc** [comment]
The PRP timing of exponent 113926861 is 847 us/it.

**Comment by selroc** [comment]
The PRP timing of exponent 111754183 is 841 us/it.

**Comment by selroc** [comment]
Weird enough, same situation but changed mainboard and RAM:  
The PRP timing of exponent 111754207 is ~ 834 us/it.

**Reply by selroc** [reply]
> Weird enough, same situation but changed mainboard and RAM: The PRP timing of exponent 111754207 is ~ 834 us/it.

After 3 hours: 817 us/it ~ 837 us/it !!!

**Comment by selroc** [comment]
Improvement with ROCm 5.1

Exponent 111755179
After 3 hours: 811 us/it

**Comment by selroc** [comment]
Slight improvement with my build of latest commit. ROCm 5.1.1, 802~806 us/it.
Exponent: 110555821
